notifications_utils/__init__.py

```diff
@@ -17,3 +17,8 @@
     r'(^|\s)\S+@\S+(\s|$)',
     flags=re.MULTILINE,
 )
+
+# The magic sequence is a ‘unique’ series of characters which we temporarily insert
+# and then later remove when performing tricky formatting operations
+MAGIC_SEQUENCE = "🇬🇧🐦✉️"
```

Comment:
Could this go in the new markdown module?

While the above code sets a precedent, this repo does lots of different things and yet these two variables are only used for a relatively small subset of them; they don't strike me as particularly "global" and are actually specific to the markdown transformations we perform.